pyproject.toml

```diff
@@ -31,7 +31,6 @@ lint = [
 ]
 unit = [
     "coverage[toml]==7.9.1; python_version > '3.8'",
-    "pytest>=8.3.5; python_version < '3.9'",
```

Comment:
We are running the tests on noble mostly, and 3.8 is EOL.

Reply:
Missed `unit = [` . Not topical. Lib supports focal.